protostar/testing/cairo_cheatcodes/cairo_cheatcode.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC
+
+from protostar.starknet import HintLocal
+from protostar.starknet.cheatable_cached_state import CheatableCachedState
+from protostar.starknet.cheatable_state import CheatableStarknetState
+from protostar.starknet.cheaters import Cheaters
+from protostar.starknet.forkable_starknet import ForkableStarknet
+
+
+class CairoCheatcode(HintLocal, ABC):
+    def __init__(self, starknet: ForkableStarknet):
+        self.starknet = starknet
+
+    @property
+    def cheatable_starknet_state(self) -> CheatableStarknetState:
+        return self.starknet.cheatable_state
+
+    @property
+    def cheatable_state(self) -> CheatableCachedState:
+        return self.cheatable_starknet_state.cheatable_state
+
+    @property
+    def cheaters(self) -> Cheaters:
```

Comment:
I think using cheaters for remaining cheatcodes as well is a nice idea, this pattern will work in favour of decoupling.

Why do we need to expose cheaters from cheatcode though? I believe it should be blackbox as well

Reply:
Will move more logic into cheaters

Reply:
Done